cnxarchive/search.py

```diff
@@ -654,4 +654,4 @@ def search(query, query_type=DEFAULT_QUERY_TYPE,
             cursor.execute(statement, arguments)
             search_results = cursor.fetchall()
     # Wrap the SQL results.
-    return QueryResults(search_results, query, query_type)
+    return QueryResults(search_results, query, query_type)
```

Comment:
Is it ok to revert the change to this file?  If there are any files with only whitespace changes that are not necessary, do `git checkout cnxarchive/search.py`, so you revert the change before you commit.